src/pages/Login/Signup.styles.scss

```diff
@@ -0,0 +1,364 @@
+@import url('https://fonts.googleapis.com/css?family=Biryani&display=swap');
+
+body {
+    background: #050516;
+    overflow: hidden;
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    display: flex;
+    justify-content: center; /*centers items on the line (the x-axis by default)*/
+    align-items: center; /*centers items on the cross-axis (y by default)*/
+
+    .signup-component {
+        background: #0d4e67;
+        border-radius: 4px;
+        width: 500px;
+        height: 500px;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+    }
+
+    .signup {
+        display: flex;
+        flex-wrap: wrap;
+        width: 400px;
+        height: 400px;
+
+        .headline {
+            display: flex;
+            flex-wrap: wrap;
+            height: 20%;
+            justify-content: flex-start;
+
+            .headline1 {
+                font-style: normal;
+                font-weight: normal;
+                font-size: 25px;
+                line-height: 140%;
+                letter-spacing: 0.1em;
+                color: #ffffff;
+                width: 100%;
+                height: 50%;
+            }
+            .sub-headline {
+                font-style: normal;
```

Comment:
By default font-style is normal and font-weight is normal. You don't need to add it.